lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -23,6 +19,7 @@ class FeatureCard extends React.Component {
     this.campaigns = [];
     this.campaignsToExclude = setting('dsUser.activity.signups', []);
     this.campaignsApiPage = 1;
+    this.apiClient = new ApiClient(window.location.hostname, '/api/v1/', window.location.port);
```

Comment:
Yeah, the `port` does have a default in the ApiClass constructor and was kinda wondering if the host one should default to the `window.location.hostname` as well. Maybe even the `version` has a default too? Wasn't sure how crazy to get.

The port made the most sense for sure. If it's not present it's passed as an empty string to the constructor and then I do a check there to normalize.